web/src/RootUser.jsx

```diff
@@ -57,32 +61,50 @@ export default function RootUser() {
 
   const accept = async () => {
     // TODO: handle errors
-    if (rootPassword !== hiddenPassword && rootPassword !== "") {
+    if (rootPassword !== hiddenPassword) {
       await client.users.setRootPassword(rootPassword);
     }
+    const remembered_password = rootPassword === "" ? "" : hiddenPassword;
+    client.users.setRootSSHKey(SSHKey);
     // TODO use signals instead
-    dispatch({ type: "ACCEPT", payload: { rootPassword: hiddenPassword } });
+    dispatch({ type: "ACCEPT", payload: { rootPassword: remembered_password } });
```

Comment:
NOTE: I got rid of the remembered_ password in another PR. From my point of view, it's not necessary to show a fake remembered password to the user. Displaying the form for setting the new password is enough. We can discuss it later.